crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -143,7 +143,7 @@ def adminrepocheck
             repos: {}
           }
           ret[:openstack][:repos][admin_architecture.to_sym] = {
-            missing: ["SUSE OpenStack Cloud 7"]
+            missing: ["SUSE-OpenStack-Cloud-7"]
```

Comment:
UI doesn't currently check `missing` anywhere but int the main functionality it expects "Pool" and "Updates" exactly as in your comment.